media-collection/STANDALONE.md

```diff
@@ -0,0 +1,182 @@
+# Media Data Collection
+
+## Overview
+
+## Configuration
+
+To be able to use Media Edge Data Collection via WEB SDK, it has to be configured via `configure` command. The `configure`
+command call would look something like this:
+
+```javascript
+ alloy("configure", {
+  mediaCollection: {
+    channel: "video channel",
+    playerName: "test player",
+    version: "alloy 2.16.0"
+  },
+  ...
+});
+```
+
+## Create a Media Session
+
+There are few ways to implement Media Edge Data Collection:
+
+- `automatic` - in this setup Web SDK takes care of the session
+- `non-automatic` - in this setup customer is responsible for creating and populating the `xdm.mediaCollection` object.
+
+### Automatic Media Session Handling
+
+When using `automatic` setup for Media Edge Data Collection, WEB SDK will take care of session ID handling and keeping track of
+player stater. In order to be able to successfully use this setup WEB SDK needs to know a few details when dealing with
+player events. To make sure that all this can be successfully wired WEB SDK exposes a new command named `createMediaSession`.
```

Comment:
Does the `Create Media Session` event replace the `Play` event, or do we still need a `play` Rule after we create the session?

Reply:
we still need play

Reply:
we create media session, to aquire a session ID from Media Edge.